crates/bevy_sprite/src/sprite.rs

```diff
@@ -23,6 +25,26 @@ pub struct Sprite {
     pub anchor: Anchor,
 }
 
+#[derive(Component, Debug, Default, Clone, Reflect)]
+#[reflect(Component, Default)]
+pub enum SpriteScaleMode {
```

Comment:
IMO we should call this `ImageScaleMode`, since we're intending to support non-sprite images eventually.

Reply:
Right !

Reply:
`bevy_render` feels like the correct home, and is a shared dependency.

Reply:
`bevy_sprite` is also a dependency of `bevy_ui`. and I'm not sure `bevy_render` is the right place, it feels like this abstraction is too high level for this core crate

Reply:
Yeah @superdump and @robftm can decide :) Here is fine for now though.